airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1157,6 +1157,10 @@
   icon: mssql.svg
   sourceType: database
   releaseStage: alpha
+  allowedHosts:
+    hosts:
+      - "${host}"
+      - "${tunnel_method.tunnel_host}"
```

Comment:
What happens if there's no tunnel. i.e not tunnel host?

Reply:
It is not used when creating the source docker container (connector framework handles it). 
This source [connection](https://cloud.airbyte.com/workspaces/d94b6349-97d8-4887-8e6c-1bc681680ffe/source/f437e661-408e-4f1f-b43f-3caa34e0f243/settings) here is setup _without_ a tunnel host, and these are the logs (note only the `host` IP address is sent with allowedHosts)

The above connection is setup for Postgres and should work the same way for MySQL and MS-SQL because of common wiring. 

`Attempting to start pod = source-postgres-strict-encrypt-read-1492182-0-fyjut for airbyte/source-postgres-strict-encrypt:2.0.0 with resources io.airbyte.config.ResourceRequirements@446d7669[cpuRequest=1,cpuLimit=2,memoryRequest=2Gi,memoryLimit=2Gi] and allowedHosts io.airbyte.config.AllowedHosts@43e6f05f[hosts=[34.23.30.27, *.datadoghq.com, *.datadoghq.eu, *.sentry.io],additionalProperties={}]`

Reply:
Sounds good - Only making sure this won't blow up in such case.